src/main/java/com/_up/megastore/controllers/interfaces/IProductController.java

```diff
@@ -18,4 +21,16 @@ ProductResponse saveProduct(
             @RequestPart MultipartFile multipartFile
     );
 
+    @GetMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    ProductResponse findProductById(@PathVariable UUID id);
+
+    @GetMapping()
+    @ResponseStatus(HttpStatus.OK)
+    Page<ProductResponse> findProductsByPages(
```

Comment:
Aca no hace falta que agreguemos el sufijo "ByPages" porque si bien obtenemos una pagina en particular, no estamos buscando los productos por pagina especificamente, solo es una técnica para evitar la obtención de muchos datos en la base de datos, yo lo dejaría así nomas

```suggestion
    Page<ProductResponse> findProducts(
```

Reply:
Incluso, es mejor que especifiquemos el "byName" porque sí es un filtro por el cual vamos a estar buscando los productos (no lo agregaria tampoco ya que es opcional)